### ISSUE TITLE
Minor typo in the CF description

### ISSUE THREAD

**Issue by SplendidAlakey** [TelepathicGrunt/NoNullProcessors#1]
"As far as we know, 1.16.5 does not have this issue and but 1.18+ does." I assume it should be "As far as we know, 1.16.5 does not have this issue, but 1.17+ does."

**Comment by TelepathicGrunt** [comment]
It wasn’t a typo but letting users know that 1.17 is the first to have the issue and it continued to persist in 1.18. But I’ll reword it to be more clear

**Comment by SplendidAlakey** [comment]
The "and but" is the typo I meant.

**Comment by TelepathicGrunt** [comment]
O. Fixed now anyway lol. Thank you
